index.html

```diff
@@ -529,7 +529,7 @@ <h2>OpenID Connect</h2>
         </section>
       </section>
     <section class="normative">
-      <h2>JSON Web Token Header Parameters</h2>
+      <h2>Protected Header Parameters</h2>
```

Comment:
Adjusted this language, because sd-jwt is not JWT, but also uses protected headers.